internal/resources/app_endpoint_oidc_schema.go

```diff
@@ -0,0 +1,87 @@
+package resources
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+)
+
+func AppEndpointOidcProviderSchema() schema.Schema {
+	return schema.Schema{
+		MarkdownDescription: "App Endpoint OpenID Connect Provider Resource",
+		Attributes: map[string]schema.Attribute{
+			"app_endpoint_name": schema.StringAttribute{
+				Required: true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.RequiresReplaceIfConfigured(),
+				},
+				MarkdownDescription: `The name of the App Endpoint. Requires replacement if changed.`,
+			},
+			"app_service_id": schema.StringAttribute{
+				Required: true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.RequiresReplaceIfConfigured(),
+				},
+				MarkdownDescription: `The GUID4 ID of the App Service. Requires replacement if changed.`,
+			},
+			"client_id": schema.StringAttribute{
+				Required:            true,
+				MarkdownDescription: `The OpenID Connect provider client ID.`,
+			},
+			"cluster_id": schema.StringAttribute{
+				Required: true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.RequiresReplaceIfConfigured(),
```

Comment:
`RequiresReplaceIfConfigured` is not needed since this is a `required` attribute